src/http-api.ts

```diff
@@ -63,6 +63,9 @@ export class HttpApi {
     );
   }
 
+  /* Handler for setting up Express API routes
+   * API routes for `/`, `/health`, `/metrics`, `/:did/records/:id`
```

Comment:
I don't think we should enumerate all endpoints here. Adding a new endpoint shouldn't require updating this comment. Maybe this could be documented as:

> setupRoutes configures the http server's request handlers